hadoop-ozone/dist/src/main/smoketest/omha/om-leader-transfer.robot

```diff
@@ -44,30 +44,135 @@ Get One OM Follower Node
                             LOG                     Follower OM: ${followerOM}
     [return]                ${followerOM}
 
+Get OM Leader and One Follower Node
+	${follower_node} =    Get One OM Follower Node
+	${leader_node} =      Get OM Leader Node
+	[Return]              ${leader_node}    ${follower_node}
```

Comment:
This invokes `ozone admin om roles` twice.  We can improve it by making the following changes:

1. Create a new keyword (`Get OM Nodes` or similar) to return all OM nodes in (leader, follower1, follower2) order.  This new keyword can reuse the logic currently duplicated between `Get OM Leader Node` and `Get One OM Follower Node`.
2. Change `Get OM Leader Node` and `Get One OM Follower Node` to call the new keyword and return the right node, removing duplicated logic.
3. Implement the new `Get OM Leader and One Follower Node` by calling the new keyword.

Reply:
Updated the PR with the suggestions. Please review. Thanks!